src/main/java/com/google/impactdashboard/data/project/ProjectGraphData.java

```diff
@@ -9,8 +9,8 @@
 public abstract class ProjectGraphData {
 
   public abstract String getProjectId();
-  public abstract Map<Long, Integer> getNumberIAMBindingsOnDate();
-  public abstract Map<Long, Recommendation> getRecommendationsAppliedOnDate();
+  public abstract Map<Long, Integer> getDateToNumberIAMBindings();
+  public abstract Map<Long, Recommendation> getDateToRecommendationAppliedTaken();
```

Comment:
nit: Should be `getDateToRecommendationTaken` per design doc.

Reply:
I thought I had changed it to the correct variable. It's fixed now.